sailpoint-bytewax.dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM 406205545357.dkr.ecr.us-east-1.amazonaws.com/sailpoint/python3.10:1 AS build
```

Comment:
let's add comment with a path to the original Dockerfile for posterity   `sdk/python/feast/infra/materialization/contrib/bytewax/Dockerfile`